cli/dts/lib.deno.unstable.d.ts

```diff
@@ -761,6 +761,49 @@ declare namespace Deno {
     mtime: number | Date,
   ): Promise<void>;
 
+  /** Spawns new subprocess. RunOptions must contain at a minimum the `opt.cmd`,
```

Comment:
I suggest to revert this bit and leave `lib.deno.ns.d.ts` section for `Deno.run` as it was

Reply:
As I have [mentioned](https://github.com/denoland/deno/pull/15441#discussion_r942670968), this just follows existing examples:

https://github.com/denoland/deno/blob/eab66a9f4da4e166df6f2831cd0141566cd7fc45/ext/net/lib.deno_net.d.ts#L81-L90

https://github.com/denoland/deno/blob/eab66a9f4da4e166df6f2831cd0141566cd7fc45/cli/dts/lib.deno.unstable.d.ts#L964-L978

At the very least, these should be marked as `**UNSTABLE**` in the documentation.